modules/gateway-task/outputs.tf

```diff
@@ -22,3 +22,10 @@ output "task_tags" {
   value = aws_ecs_task_definition.this.tags_all
 }
 
```

Comment:
We could maybe use an output for the security group id that is created for the LB, if it is created by this module?